src/WebJobs.Script/Workers/Rpc/RpcWorkerDescription.cs

```diff
@@ -73,6 +73,8 @@ public List<string> Extensions
             }
         }
 
+        public override bool UseStdErrorStreamForErrorsOnly { get; set; } = false;
```

Comment:
What's the behavior with placeholders? For example, if the placeholder version has this property set to "true", but a user overrides it as false? I don't think this is a super valid scenario, but just wondering!

Reply:
This is a config property coming from worker.config.json -- not user controlled. Let me know if I misunderstood question.